app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    @like = @post.likes.build(user: @user)
+
+    if @like.save
+      redirect_to user_posts_path(@user)
+    else
+      redirect_to user_posts_path(@user), alert: 'Error creating like.'
```

Comment:
[OPTIONAL] It could be better if the like is created on behalf of the currently logged in user (current_user) of the app. eg:

```rb

    def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
	  ...

  end
```

- Please fix this issue and don't skip this issue because this will cause some problems in future so kindly fix it :slightly_smiling_face: